gammapy/modeling/models/spectral.py

```diff
@@ -213,6 +213,29 @@ def evaluate_error(self, energy, epsilon=1e-4):
         """
         return self._propagate_error(epsilon=epsilon, fct=self, energy=energy)
 
+    def pivot_energy(self, energy):
+        """Return the pivot energy for a given spectral model.
+        The pivot energy is the narrowest point of the butterfly.
```

Comment:
The `butterfly' might not always have a shape of a butterfly.... I am wondering whether this is not too much into our jargon. Maybe one can just say energy with the smallest statistical error..
In the same manner, is `pivot_energy' a standard statistical word?

Reply:
Okay, I agree with the use of the word butterfly. 
As for pivot energy, this is what the already defined function in `PowerLawSpectralModel` was called. So I kept it this way, open to suggestions. It has also been called the 'decorrelation energy'